contents/docs/cdp/batch-exports/index.md

```diff
@@ -32,12 +33,78 @@ A batch export is executed in batch runs depending on the configured frequency.
 
 As an example, creating a batch export of events with daily frequency today will schedule the first batch run to start right as tomorrow begins. Thus, the data exported are events that PostHog received from 00:00:00 until 23:59:59 of today.
 
-Batch runs are configured on the [exports page](https://app.posthog.com/exports) in your PostHog instance.
-
 > **Note:** When deciding which data falls within the bounds of a specific batch run, we look at the time when the data landed in PostHog's ClickHouse database. This means that network and processing delays can make an event that was sent within the bounds of a batch run fall in a future run.
 
-## How does this differ from the old export system?
+### Tracking progress
+
+On each batch export view we will be presented with a list of the latest runs executed:
+
+![batch export runs](../../../images/docs/batch-exports/batch-exports-runs.png)
+
+Each run has an indicator of the state (which can be either "Starting", "Running", "Failed", or "Completed"), the boundaries for the data exported (data interval start and data interval end), when the run actually started, and the option of retrying a specific run.
+
+
+## Historical exports
+
+Batch exports can export data that landed in PostHog before the batch export was created, known as historical data. For this, no new batch export has to be created: A batch export already knows the destination where we wish to export historical data, it only requires the boundaries for the historical export, in other words a start and an end date.
+
+A "Create historic export" button can be found in the UI:
+
+![batch exports ui](../../../images/docs/batch-exports/batch-exports-ui.png)
+
+Which will let users input the start and end date of the historical export:
+
+![create historic export](../../../images/docs/batch-exports/create-historic-export.png)
+
+Immediately afterwards, the historical export runs that fall within the bounds selected will be scheduled.
```

Comment:
```suggestion
Immediately afterwards, a historical export runs for the bounds selected is scheduled.
```

Reply:
For example: An hourly historical export from 2023-08-22 to 2023-08-23 would create 24 runs (1 for each our of the historical export).

Reply:
Ahh, I get it, can just move wording to the present, unless scheduling doesn't happen right away.
```suggestion
Immediately afterwards, the historical export runs that fall within the bounds selected are scheduled.
```

Reply:
It does happen right away (with some minimal lag as the services talk to each other).